AnkiDroid/src/main/java/com/ichi2/libanki/DecksV16.kt

```diff
@@ -920,3 +921,11 @@ class DecksV16(private val col: CollectionV16) :
 fun CollectionV16.getDeckNamesRaw(input: ByteArray): ByteArray {
     return backend.getDeckNamesRaw(input)
 }
+
+/**
+ * Gets the filtered deck with given [did]
+ * or creates a new one if [did] = 0
+ */
+fun CollectionV16.getOrCreateFilteredDeck(did: DeckId = 0): FilteredDeckForUpdate {
+    return backend.getOrCreateFilteredDeck(did)
```

Comment:
Sorry, I wasn't clear - I meant that when calling backend methods, explicit parameter names should be used to reduce the chances of accidental breakages in future backend updates, eg

```suggestion
    return backend.getOrCreateFilteredDeck(did=did)
```

Reply:
Oh, makes sense. Idk if Kotlin has something to force using naming arguments, but if yes, it would be good for the future